app/src/components/fields/fieldErrorHint/fieldErrorHint.jsx

```diff
@@ -54,6 +54,10 @@ const messages = defineMessages({
     id: 'RegistrationForm.emailDuplicateHint',
     defaultMessage: 'Entered email already exists in the system.',
   },
+  validEmailHint: {
+    id: 'ServiceModal.validEmailHint',
+    defaultMessage: 'Please enter valid email',
+  },
```

Comment:
Can we reuse [this](https://github.com/reportportal/service-ui/pull/3841/files#diff-3f5907b3664c3818d623ad7efeccf978a02ba4268cdefba97e727e473ea22588R41) message instead?
![image](https://github.com/reportportal/service-ui/assets/31861805/4710c4b1-09e3-4dfc-91ce-38055bb381fb)
Please discuss with BA